theme/n21/views/help/mail-card.hbs

```diff
@@ -1,5 +1,5 @@
 <p>Manche Anliegen brauchen mehr Möglichkeiten, als die Felder der Kontaktfunktion zulassen. Schreib uns eine E-Mail, wenn du mit uns ins Gespräch kommen willst:</p>
 <div id="email-addresses">
-    <a href="mailto:feedback@hpi-schul-cloud.de" target="_blank" rel="noopener">feedback@hpi-schul-cloud.de</a> ans HPI für Ideen, Kritik und Wünsche rund um die Cloud selbst. <br>
+    <a href="mailto:nbc-wunsch@netz-21.de" target="_blank" rel="noopener">nbc-wunsch@netz-21.de</a> ans HPI für Ideen, Kritik und Wünsche rund um die Cloud selbst. <br>
```

Comment:
shouldn't it be nbc-support@netz-21.de ?

Reply:
this one is below ;) they should be different